moduleframework/mtf.yaml

```diff
@@ -0,0 +1,104 @@
+# default module type in case you don't wath everytime use MODULE=docker or define it in config.yaml
+default_module: docker
+
+# openshift specific section
+openshift:
+  init_wait: 50
```

Comment:
Add more documentation

Reply:
I'm not able to add there documenttation, because you wrote this part and I've just moved these constants out of code to this file without knowledge what this does mean